exercise_2/Graph.cpp

```diff
@@ -0,0 +1,69 @@
+#include "Graph.h"
+#include <iostream>
+#include <vector>
+#include "WeightMatrix.h"
+
+Graph::Graph(const int& vertices, WeightMatrix& m)
+{
+	this->vertices = vertices;
+	this->matrix = m;
+}
+
+Graph::~Graph() = default;
+
+int Graph::minIndex(std::vector<int>& v, std::vector<bool>& visited)
```

Comment:
Function name should describe some activity so it should be created on verbs basis. E.g. setMinIndex(), getMinIndex(), calculteSomething etc.